File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1181_basico.js
import { question } from "readline-sync";

function main() {
    let matriz = [];
    let ordem = 12;
    let linhaDesejada = get_Number();
    let operacao = get_Text().toUpperCase();
    let s = 0;

    if(linhaDesejada >= 0 && linhaDesejada <= 11) {
        for(let i = 0; i < ordem; i++) {
            let linha = [];
    
            for(let j = 0; j < ordem; j++) {
                let valor = get_Number();
                linha.push(valor);
    
                if(i === linhaDesejada) {
                    s += valor;
                }
    
            }
            matriz.push(linha);
    
        }

    }
    let m = s / ordem;
    
    if(operacao === `S`) {
        print(`${s.toFixed(1)}`);

    }else if(operacao === `M`) {
        print(`${m.toFixed(1)}`);

    }

}

function get_Number() {
    return Number(question());
}

function get_Text() {
    return question();
}

function print(texto) {
    return console.log(texto);
}

main();

File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1181_basico.py
def main():
    matriz = []
    ordem = 12
    linha_Desejada = get_Number()
    operacao = get_Text().upper()
    s = 0

    for i in range(ordem):
        linha = []

        for j in range(ordem):
            valor = get_Number()
            linha.append(valor)

            if (i == linha_Desejada):
                s += valor

        matriz.append(linha)
    
    m = s / ordem

    if (operacao == 'S'):
        print(f"{s:.1f}")
    elif (operacao == 'M'):
        print(f"{m:.1f}")

def get_Number():
    return float(input().strip())

def get_Text():
    return input().strip()

main()